src/upnp_xml.cc

```diff
@@ -424,33 +431,30 @@ std::string UpnpXMLBuilder::renderOneResource(const std::string& virtualURL, con
 bool UpnpXMLBuilder::renderItemImage(const std::string& virtualURL, const std::shared_ptr<CdsItem>& item, std::string& url)
 {
     bool artAdded = false;
-    int realCount = 0;
-    for (auto&& res : item->getResources()) {
+    auto orderedResources = getOrderedResources(item);
+    for (auto&& res : orderedResources) {
```

Comment:
This looks like it can be converted to std::any_of

Reply:
I don't see how any_of can work here, find_if does

Reply:
Yeah, my bad. Need the iterator.